viz_config.yml

```diff
@@ -14,6 +14,11 @@ gage_norm_col: "#4BA3C3"
 hurricane_col: "orange"
 n_intro_frames: 20 # number of frames in the intro animation linking gages to sparklines
 frame_delay_cs: 8 # delay between frames of the gif animation. cs = centiseconds = hundredths of a second
+sparks:
+  xleft: 0.85 # as percentage of figure width
+  xright: 1
+  ybottom: 0.30 # as percentage of figure height
+  ytop: 1
```

Comment:
cool, and i think we can work this into jake's `image_placement` config element (https://github.com/USGS-VIZLAB/vizstorm-GIF/pull/76/files#diff-0c27760a36bf7df1b67ae0e2f1de32e6R20) in a future PR

Reply:
(note that his image_placement organization has changed since i wrote that)